src/store/modules/my_profile.js

```diff
@@ -22,6 +22,9 @@ export default {
     setRelayData (state, relayData) {
       state.profile = relayData.profile
       state.inbox = relayData.inbox
+    },
```

Comment:
This looks unrelated to the PR topic and should probably be a separate PR.  Thank you for fixing this though!

Reply:
This is odd, I think I got this from another commit?